interactive_ai/services/inference_gateway/chart/templates/deployment.yaml

```diff
@@ -47,7 +47,11 @@ spec:
           {{- toYaml .Values.securityContext | nindent 12 }}
           env:
             - name: GOMAXPROCS
-              value: "{{ .Values.resources.requests.cpu }}"
+              {{- if (eq .Values.global.installation_profile "lite") }}
+              value: "{{ .Values.liteResources.requests.cpu }}"
+              {{- else }}
+              value: "{{ .Values.standardResources.requests.cpu }}"
+              {{- end }}
```

Comment:
This env var does not work with fractional CPU.  In this scenario, it should be set to 1 for lite profile.